lib/ansible/modules/cloud/amazon/ecs_service.py

```diff
@@ -438,12 +450,15 @@ def delete_service(self, service, cluster=None):
         return self.ecs.delete_service(cluster=cluster, service=service)
 
     def ecs_api_handles_network_configuration(self):
-        from distutils.version import LooseVersion
         # There doesn't seem to be a nice way to inspect botocore to look
         # for attributes (and networkConfiguration is not an explicit argument
         # to e.g. ecs.run_task, it's just passed as a keyword argument)
         return LooseVersion(botocore.__version__) >= LooseVersion('1.7.44')
 
+    def health_check_setable(self, params):
+        load_balancers = params.get('loadBalancers', [])
+        # check if botocore (and thus boto3) is new enough for using the healthCheckGracePeriodSeconds parameter
+        return len(load_balancers) > 0  and LooseVersion(botocore.__version__) >= LooseVersion('1.9.0')
```

Comment:
AnsibleAWSModule has a function to check boto3/botocore versions you can use here. 
```suggestion
        return len(load_balancers) > 0  and self.module.botocore_at_least('1.9.0')
```